prowler/lib/check/models.py

```diff
@@ -38,7 +39,7 @@ class Check_Metadata_Model(BaseModel):
     CheckID: str
     CheckTitle: str
     CheckType: list[str]
-    CheckAlias: list[str] = []
+    CheckAlias: Optional[list[str]] = []
```

Comment:
You don't need the `= []` when the type is `Optional` since underneath it will default to `None`

Reply:
If I don't add it, it fails since I need it to be an empty list by default, not None.

Reply:
In that case remove the optional, since it won't be `None`.